### PR TITLE
Sync live branch to dev branch

### PR DESCRIPTION
**Pull request by TaitKaito**
Syncing live to dev so they share the same baseline, lets crack some heads

### CODE REVIEW

File: Weapon Mods/Development/TaitMod_Fletcher_Dev/Data/CubeBlocks/BarbetteSet.sbc
Comment (InvalidArgument3) [inline Weapon Mods/Development/TaitMod_Fletcher_Dev/Data/CubeBlocks/BarbetteSet.sbc:1]: these monstrosities are going to need actual points

File: Weapon Mods/Development/TaitMod_Fletcher_Dev/Data/CubeBlocks/Mk25 Rangefinder.sbc
Comment (InvalidArgument3) [inline Weapon Mods/Development/TaitMod_Fletcher_Dev/Data/CubeBlocks/Mk25 Rangefinder.sbc:1]: this one had its HP set to sane levels with an emergency push in #63

**Review by InvalidArgument3 — CHANGES_REQUESTED** [review]
is this how reviewing works?

**Comment by TaitKaito** [comment]
I was about to start doing my todo list but got shouted away by family...

**Comment by TaitKaito** [comment]
Full Changelog in written format:
- Prinz Eugen fragment is being called correctly
- Total impulse death!!!!
- Rangefinder is reasonable (Thnx invalid)
- Prinz Eugen is no longer super integrity (140k - 83k)
- Fletcher/Benson/Atlanta No longer has healing shell 
- Neptune Rework:
	Ammo
		Stages: 3 - 2
		Torpedo Speed: 150 - 200
		EndOfLife(actual damage): 40k - 30k
		Falloff: pooled
		Radius: 12.5
		depth: 10f
		Inaccuracy: 0.5
		Aggressiveness: 10
		MaxLateralthrust: 20
	Launcher
		RateOfFire: 2500 - 500
		Keeping 1 minute reload time
		AimingTolerance: 2 - 5
		GoHomeToReload: True - False
		Climbing cost: 20%
		BaseBP: 500
-Barbette Set
	Barbette durability synced as if they were buster conveyors (25, 9, 1 based on 5x5, 3x3 and 1x1 respectivly)
	5x5:
		BP: 100
	3x3:
		BP: 36
	1x1:
		BP: 4
- 40mm Flak Quad:
	Integrity fix: 23k - 9k

**Pull request merged**